src/views/ward-leader.vue

```diff
@@ -293,6 +293,69 @@ export default {
     ]),
     sampleBallotFormPrefilled () {
       return `${SAMPLE_BALLOT_FORM}?ward=${this.ward}&party=${this.party}`
+    },
+    allCommitteePersons: function () {
```

Comment:
very minor, you can write this in the short hand style like the one above:

`    allCommitteePersons() {`